src/View/ViewableData.php

```diff
@@ -678,33 +615,12 @@ public function getXMLValues($fields)
         return $result;
     }
 
-    // ITERATOR SUPPORT ------------------------------------------------------------------------------------------------
-
-    /**
-     * Return a single-item iterator so you can iterate over the fields of a single record.
-     *
-     * This is useful so you can use a single record inside a <% control %> block in a template - and then use
-     * to access individual fields on this object.
-     *
-     * @deprecated 5.2.0 Will be removed without equivalent functionality
-     *
-     * @return ArrayIterator
-     */
-    public function getIterator(): Traversable
-    {
-        Deprecation::notice('5.2.0', 'Will be removed without equivalent functionality');
-        return new ArrayIterator([$this]);
-    }
```

Comment:
Unrelated but seems sensible to just get this out of the way now - I'll need to know that things work as expected without this method there for the refactor anyway.